### ISSUE TITLE
After open the livedesplay,mobile phone is not smooth

### ISSUE THREAD

**Issue by hesier** [divis1969/android_device_meizu_meilan2#24]

**Comment by divis1969** [comment]
Could you please provide more details on use case? Do you mean i need to enable automatic mode in settings->display->live display?

**Comment by hesier** [comment]
Yes,such as settings->display->live display->night, then mobile phone will become very not smooth, especially for the settings
